### PR TITLE
Update for DGtal 1.0

### PR DESCRIPTION
**Pull request by kerautret**
todo:
- [x] finish broken links in gallery (a little long)
- [x] link to 1.0 of tools.
- [x] various check
- [x] Link to new doc
- [x] Check listing tools of DGtalTools
- [x] Check listing tools of DGtalToolsContrib

### CODE REVIEW

File: dgtaltools.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: DGtalTools
 ---
 
 
-[DGtalTools][1] is a separate github project containing tools constructed using DGtal library. The main goal of this part is to gather simple and useful tools exploiting the structures and algorithms defined in DGtal. Note that in complement to DGtalTools, the [DGtalTools-contrib][2] project was added to gather specific research tools or prototypes. The resulting tools could be useful to:
+[DGtalTools][1] is  separate github project containing tools constructed using DGtal library. The main goal of this part is to gather simple and useful tools exploiting the structures and algorithms defined in DGtal. Note that in complement to DGtalTools, the [DGtalTools-contrib][2] project was added to gather specific research tools or prototypes. The resulting tools could be useful to:
```
Comment (dcoeurjo) [inline dgtaltools.md]: I would keep the “a”

Comment (kerautret) [reply dgtaltools.md]: oops sorry it was done to test the Markdown editor/viewer ;)

**Comment by kerautret** [comment]
@dcoeurjo thanks for the remarks, else I have finished the update that should be fine.

**Comment by dcoeurjo** [comment]
Thx done

**Pull request merged**